### ISSUE TITLE
Should we index and display <revisiondesc> ?

### ISSUE THREAD

**Issue by marlo-longley** [projectblacklight/arclight#1454]
It appears in the EAD header. We don't know if it is desired to display or if it's more of an internal thing for catalogers.

This ticket is broken out of https://github.com/projectblacklight/arclight/issues/898

**Comment by gwiedeman** [comment]
I would say yes as its transparency that's very useful for users. The ASpace PUI displays this.

Here is an example `<revisiondesc>`:
```
<revisiondesc>
	<change>
		<date>1989 July 10</date>
		<item>Encoded in EAD by Geoffrey P. Williams, Geoffrey A. Huth, and Salvatore Canino</item>
	</change>
	<change>
		<date>2001 April 17</date>
		<item>Updated by Lynn Dorwaldt</item>
	</change>
	<change>
		<date>2016</date>
		<item>Reprocessed and converted to EAD by Ben Covell</item>
	</change>
	<change>
		<date>2023</date>
		<item>Updated by Meghan Slaff</item>
	</change>
</revisiondesc>
```

**Comment by mmmmcode** [comment]
I've always viewed \<revisiondesc\> as an internal change summary. If I'm interpreting the [ead2002 tag library](https://www.loc.gov/ead/tglib/elements/revisiondesc.html) correctly, it is supposed to be used for tracking changes to the finding aid as an ead document, and not necessarily to the finding aid's intellectual content. There are other elements for tracking substantial changes to collection description or contents, like \<processinfo\>, \<separatedmaterial\>, etc.  Interested to hear how other folks use it!

**Comment by mmmmcode** [comment]
@gwiedeman Hmm, I didn't know the ASpace PUI displayed it. I'm just thinking of all the revisiondesc notes that say stuff like "fixed typo", "revised note order", "updated repository information", etc., and whether that really matters to researchers. Maybe it can at least be something you have to click through to, rather than have front-and-center?

**Comment by gwiedeman** [comment]
Yeah, this has come up at TS-DACS with https://github.com/saa-ts-dacs/dacs/issues/70, as this should be more clearly defined and there's probably a ton of variation in practice. We often end up documenting reprocessing in both revisions and processing notes which is not great.

Also, the ASpace PUI displays it _if_ its published. Though for ASpace workflows at least, if its not published it shouldn't export to the EAD.
